src/styles/blocks/page.scss

```diff
@@ -0,0 +1,36 @@
+.page {
+  scroll-behavior: smooth !important;
```

Comment:
Using `important` is a bad practice, unless you have a strong reason to do it.

Reply:
Yes, smooth doesn't work in my current version of Chrome without '!important'. I read Stackoverflow on this topic and found the following solution